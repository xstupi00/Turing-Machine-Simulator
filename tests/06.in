S a B c
S a B d
S a C a
B c K L
B c D c
B c B  
B d D c
C a D b
D c D  
D b E  
E   F  
aaa
