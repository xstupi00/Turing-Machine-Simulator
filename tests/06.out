Saaa
Caaa
Dbaa
E aa
F aa
